Sources/StripeKit/Payment Methods/Sources/SourceRoutes.swift

```diff
@@ -212,6 +217,12 @@ public struct StripeSourceRoutes: SourceRoutes {
             body["usage"] = usage
         }
         
+        if extra.count > 0 {
```

Comment:
Once we have the `source: [String: Any]?` changed this can be changed as well to 
```swift
if let source = source {
   source.forEach { body["\($0)"] = $1 }
}
```